### PR TITLE
Feature/uml init

### PR DESCRIPTION
**Pull request by zero-udo**
Initial push of an Eclipse Papyrus UML Project with first draft of a "User" Component

### CODE REVIEW

File: docs/uml/Elysian Muse/User Component.uml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        <ownedAttribute xmi:type="uml:Property" xmi:id="_Xt1S0DXSEeyghuCVBA5yaw" name="bio">
+          <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#String"/>
+        </ownedAttribute>
+        <ownedAttribute xmi:type="uml:Property" xmi:id="_Y1-moDXSEeyghuCVBA5yaw" name="birthday">
+          <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/EcorePrimitiveTypes.library.uml#EDate"/>
+        </ownedAttribute>
+        <ownedAttribute xmi:type="uml:Property" xmi:id="_aDBokDXSEeyghuCVBA5yaw" name="validatedAuthor">
+          <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Boolean"/>
+        </ownedAttribute>
+        <ownedAttribute xmi:type="uml:Property" xmi:id="_cpMZkDXSEeyghuCVBA5yaw" name="onProbation">
```
Comment (BibiRosenberg) [inline docs/uml/Elysian Muse/User Component.uml:42]: Adding this and a dedicated 'suspended' role feels overkill? would ever either be active without the other?

Comment (zero-udo) [reply docs/uml/Elysian Muse/User Component.uml:42]: The wording suggest "onProbation" is triggered before completely suspending a user.
E.g.:
* User misbehaves the first time -> is placed on probation
* while on probation the user requires dedicated confirmation/a positive review before publicly creating content
* User misbehaves a second/multiple time -> account gets suspended for a set amount of time/permanently
or
* User completes probation without further offences and the probation is lifted


if suspension/probation periods are planned, an additional "suspentionDate"/"probationDate" attribute might come in handy.

**Review by BibiRosenberg — APPROVED** [review]

File: docs/uml/Elysian Muse/User Component.uml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#Boolean"/>
+        </ownedAttribute>
+        <ownedAttribute xmi:type="uml:Property" xmi:id="_eXV9cDXSEeyghuCVBA5yaw" name="settings" type="_vGsPoDXSEeyghuCVBA5yaw"/>
+      </packagedElement>
+      <packagedElement xmi:type="uml:Enumeration" xmi:id="_x2x_MDWyEeyghuCVBA5yaw" name="Role">
+        <ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="_1FL_cDWzEeyghuCVBA5yaw" name="ADMINISTRATOR"/>
+        <ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="_2oSw8DWzEeyghuCVBA5yaw" name="MODERATOR"/>
+        <ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="_3JO60DWzEeyghuCVBA5yaw" name="USER"/>
```
Comment (BibiRosenberg) [inline docs/uml/Elysian Muse/User Component.uml]: please rename to MEMBER

File: docs/uml/Elysian Muse/User Component.uml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        </ownedAttribute>
+        <ownedAttribute xmi:type="uml:Property" xmi:id="_7xwTMDW1EeyghuCVBA5yaw" name="blob">
+          <type xmi:type="uml:PrimitiveType" href="pathmap://UML_LIBRARIES/EcorePrimitiveTypes.library.uml#EByteObject"/>
+        </ownedAttribute>
+      </packagedElement>
+      <packagedElement xmi:type="uml:Class" xmi:id="_vGsPoDXSEeyghuCVBA5yaw" name="UserSettings"/>
+      <packagedElement xmi:type="uml:Enumeration" xmi:id="_APREsDXUEeyghuCVBA5yaw" name="UserTags">
+        <ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="_B4-0gDXUEeyghuCVBA5yaw" name="BETA"/>
+        <ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="_E3qzMDXUEeyghuCVBA5yaw" name="ARTIST"/>
+        <ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="_H-A5EDXUEeyghuCVBA5yaw" name="AUTHOR"/>
```
Comment (BibiRosenberg) [inline docs/uml/Elysian Muse/User Component.uml:101]: please add COMMENTER tag

File: docs/uml/Elysian Muse/User Component.uml [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        <ownedAttribute xmi:type="uml:Property" xmi:id="_P_nowDW4EeyghuCVBA5yaw" name="userRepository" type="_22-JUDW3EeyghuCVBA5yaw"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_3eN6EDW4EeyghuCVBA5yaw" name="createUser"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_5B5TUDW4EeyghuCVBA5yaw" name="suspendUser"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_6Inq0DW4EeyghuCVBA5yaw" name="changeRole"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_8EgvwDW4EeyghuCVBA5yaw" name="changeAvatar"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_AgNfADW5EeyghuCVBA5yaw" name="getById"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_Bbh3wDW5EeyghuCVBA5yaw" name="getAll"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_CG4h4DW5EeyghuCVBA5yaw" name="getAllWithRole"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_GJLSoDW5EeyghuCVBA5yaw" name="addTagToUser"/>
+        <ownedOperation xmi:type="uml:Operation" xmi:id="_HDjQMDW5EeyghuCVBA5yaw" name="removeTagFromUser"/>
```
Comment (BibiRosenberg) [inline docs/uml/Elysian Muse/User Component.uml]: please add
- deleteUser
- updateUserSettings

File: docs/uml/Elysian Muse/User Component.uml (hunk 2 above)
Comment (zero-udo) [reply docs/uml/Elysian Muse/User Component.uml]: changed

File: docs/uml/Elysian Muse/User Component.uml (hunk 3 above)
Comment (zero-udo) [reply docs/uml/Elysian Muse/User Component.uml:101]: added

File: docs/uml/Elysian Muse/User Component.uml (hunk 4 above)
Comment (zero-udo) [reply docs/uml/Elysian Muse/User Component.uml]: added

**Comment by zero-udo** [comment]
made requested changes - ready for re-review

File: docs/uml/Elysian Muse/User Component.uml [hunk 5]
```diff
@@ -97,6 +97,7 @@
       <packagedElement xmi:type="uml:Enumeration" xmi:id="_APREsDXUEeyghuCVBA5yaw" name="UserTag">
         <ownedLiteral xmi:type="uml:EnumerationLiteral" xmi:id="_B4-0gDXUEeyghuCVBA5yaw" name="BETA"/>
```
Comment (BibiRosenberg) [inline docs/uml/Elysian Muse/User Component.uml:98]: Please add BETA_INACTIVE, too

File: docs/uml/Elysian Muse/User Component.uml [hunk 6]
```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<uml:Model xmi:version="20131001" xmlns:xmi="http://www.omg.org/spec/XMI/20131001" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" xmlns:uml="http://www.eclipse.org/uml2/5.0.0/UML" xmi:id="_2tOmMDWxEeyghuCVBA5yaw" name="User Component">
+  <packageImport xmi:type="uml:PackageImport" xmi:id="_2tfr8DWxEeyghuCVBA5yaw">
+    <importedPackage xmi:type="uml:Model" href="pathmap://UML_LIBRARIES/UMLPrimitiveTypes.library.uml#_0"/>
+  </packageImport>
+  <packagedElement xmi:type="uml:Package" xmi:id="_nLb20DWyEeyghuCVBA5yaw" name="org.dark-solace.muse.user">
```
Comment (BibiRosenberg) [inline docs/uml/Elysian Muse/User Component.uml:6]: We should probabaly settle on one of the two options, either call it 'Elysian muse' or just 'muse' like you did here.
I'm ok with using the shorter version without spaces

**Comment by BibiRosenberg** [comment]
> made requested changes - ready for re-review

cannot aprove again, cannot remove approval, please merge when ready

**Pull request merged**